test/crawler_worker/keyword/crawler_worker_test.exs

```diff
@@ -0,0 +1,46 @@
+defmodule CrawlerWorker.Keyword.CrawlerWorkerTest do
+  use Crawler.DataCase, async: true
+
+  alias Crawler.Keyword.Keywords
+  alias CrawlerWorker.Keyword.CrawlerWorker
+
+  describe "perform/1" do
+    test "given successfully finished job, returns and stores the HTML result" do
+      keyword = insert(:keyword, name: "ios")
+      args = %{keyword_id: keyword.id}
+
+      use_cassette "crawl/ios_success" do
+        assert perform_job(CrawlerWorker, args) == :ok
+
+        stored_keyword = Keywords.get_keyword_by_id(keyword.id)
+
+        assert stored_keyword.status == :completed
+        assert is_binary(stored_keyword.html)
+      end
+    end
+
+    test "given status_code is 400, set keyword status failed" do
+      expect(HTTPoison, :get, fn _, _ -> {:ok, %{status_code: 400}} end)
+      keyword = insert(:keyword)
+      args = %{keyword_id: keyword.id}
+
+      assert perform_job(CrawlerWorker, args) == {:error, 400}
+
+      stored_keyword = Keywords.get_keyword_by_id(keyword.id)
+
+      assert stored_keyword.status == :failed
```

Comment:
Let's check in tests that for failed response we didn't store anything inside `html` field

Reply:
Checked in [0eb2154](https://github.com/vnntsu/elixir-ic/pull/39/commits/0eb21542fa957bf89fc373a9e127583cd95edb5d)